CoffeeChat/Controller/DemographicsViewController.swift

```diff
@@ -0,0 +1,166 @@
+        hometownSearchField?.clearButtonMode = .never
+        hometownSearchBar.setImage(UIImage(), for: .search, state: .normal)
+        hometownSearchBar.layer.cornerRadius = fieldsCornerRadius
+        hometownSearchBar.searchTextPositionAdjustment = UIOffset(horizontal: -6, vertical: 0)
+        hometownSearchBar.showsCancelButton = false
+        view.addSubview(hometownSearchBar)
+
+        pronounsTextField.backgroundColor = .backgroundLightGray
+        pronounsTextField.attributedPlaceholder = NSAttributedString(string: "Pronouns", attributes: [NSAttributedString.Key.foregroundColor: UIColor.textDarkGray])
+        pronounsTextField.leftView = UIView(frame: CGRect(x: 0, y: 0, width: 12, height: 49))
+        pronounsTextField.leftViewMode = .always
+        pronounsTextField.layer.cornerRadius = fieldsCornerRadius
+        view.addSubview(pronounsTextField)
+
+        nextButton.setTitle("Next", for: .normal)
+        nextButton.setTitleColor(.textBlack, for: .normal)
+        nextButton.backgroundColor = .backgroundLightGray
+        nextButton.layer.cornerRadius = 26
+        nextButton.addTarget(self, action: #selector(nextButtonPressed), for: .touchUpInside)
+        view.addSubview(nextButton)
+
+        setUpConstraints()
+    }
+
+    @objc func nextButtonPressed() {
+        delegate.nextPage(index: 1)
+    }
+
+    func setUpConstraints() {
+
+        let helloLabelHeight: CGFloat = 30
+        let helloLabelSpacing: CGFloat = 100
+        let nextBottomPadding: CGFloat = 90
+        let nextButtonSize = CGSize(width: 225, height: 54)
+        let textFieldSidePadding: CGFloat = 40
+        let textFieldHeight: CGFloat = 49
+        let textFieldTopPadding: CGFloat = 20
+
+        helloLabel.snp.makeConstraints { make in
+            make.centerX.equalToSuperview()
+            make.height.equalTo(helloLabelHeight)
+            make.top.equalTo(view.safeAreaLayoutGuide).offset(helloLabelSpacing)
+        }
+
+        greetingLabel.snp.makeConstraints { make in
+            make.centerX.equalToSuperview()
+            make.top.equalTo(helloLabel.snp.bottom).offset(10)
+        }
+
+        classTextField.snp.makeConstraints{ make in
+            make.centerX.equalToSuperview()
+            make.top.equalTo(greetingLabel.snp.bottom).offset(149)
+            make.left.equalToSuperview().offset(textFieldSidePadding)
+            make.right.equalToSuperview().offset(-textFieldSidePadding)
+            make.height.equalTo(textFieldHeight)
+        }
+
+        majorSearchBar.snp.makeConstraints{ make in
+            make.centerX.equalToSuperview()
+            make.top.equalTo(classTextField.snp.bottom).offset(textFieldTopPadding)
+            make.left.equalToSuperview().offset(textFieldSidePadding)
+            make.right.equalToSuperview().offset(-textFieldSidePadding)
```

Comment:
the `leading, trailing, centerX` constraints here are all the same as `classTextField` so better to just do:
`make.leading.trailing.centerX.equalTo(classTextField)`